lib/src/executables/ffigen.dart

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
```

Comment:
2021. In the Dart project we usually use the year that we add the file (not the year we started the project).

Reply:
Yeah, I actually forgot to update the template.